WordPress/Classes/ViewRelated/Media/CachedAnimatedImageView.swift

```diff
@@ -221,9 +221,9 @@ public class CachedAnimatedImageView: UIImageView, GIFAnimatable {
         animatedGifData = data
         DispatchQueue.main.async() {
             self.setFrameBufferCount(self.gifPlaybackStrategy.frameBufferCount)
-            self.animate(withGIFData: data) {
```

Comment:
If fixes the this new-ish warning: https://github.com/apple/swift-evolution/blob/main/proposals/0286-forward-scan-trailing-closures.md